dipy/direction/probabilistic_direction_getter.pyx

```diff
@@ -28,7 +29,7 @@ cdef class ProbabilisticDirectionGetter(PmfGenDirectionGetter):
     set to 0 and the result is normalized.
     """
 
-    def __init__(self, pmf_gen, max_angle, sphere, pmf_threshold=.1, **kwargs):
+    def __init__(self, pmf_gen, max_angle, sphere, pmf_threshold=.1 , **kwargs):
```

Comment:
missing `*`before  `pmf_threshold=.1 `

Reply:
Adding the `*` here will provide that error we spoke about earlier. There is some particular conflict with `__init__` method for class. It passes an extra argument by itself.

Reply:
please, remove the extra space after `pmf_threshold=.1` to be back as what it was before